src/payment/data/service.js

```diff
@@ -33,32 +34,18 @@ export function configureApiService(newConfig, newApiClient) {
 }
 
 export function transformResults(data) {
-  const transformedResults = {
-    basketId: data.basket_id,
-    currency: data.currency,
-    isFreeBasket: data.is_free_basket,
-    showCouponForm: data.show_coupon_form,
-    orderTotal: Number.parseInt(data.order_total, 10),
-    summaryDiscounts: data.summary_discounts !== null ?
-      Number.parseInt(data.summary_discounts, 10) : null,
-    summaryPrice: Number.parseInt(data.summary_price, 10),
-    products: data.products.map(({
-      image_url: imageUrl,
-      title,
-      certificate_type: certificateType,
-      sku,
-      product_type: productType,
-    }) => ({
-      imageUrl, title, certificateType, sku, productType,
-    })),
-    coupons: data.coupons,
```

Comment:
Note that coupons wasn't being camel cased.  The use of camelCaseObject fixes this, so you'll see benefit_value turn to benefitValue elsewhere in this PR.